spec/models/target_spec.rb

```diff
@@ -8,6 +8,22 @@
     it { should validate_presence_of(:latitude) }
     it { should validate_presence_of(:longitude) }
     it { should validate_presence_of(:user) }
+
+    context 'when a user has 10 targets and tries to create another one' do
+      let!(:user_1) { create :user }
+      it 'should not be able to do so' do
+        targets = create_list(:target, 10, user: user_1)
+        expect do
+          user_1.targets.create(topic: 'Football', length: 500, title: 'Onceavo', latitude: -34.905446, longitude: -56.206291)
```

Comment:
Remember you can break long lines, in this case you can add a new line in the middle of the params for example.